lxc.xml

```diff
@@ -0,0 +1,590 @@
+    root to populate the new root file system. In the following steps, the
+    new &rootuser; file system will be created in
+    <replaceable>/path/to/rootfs</replaceable>.
+   </para>
+   <important>
+    <title>Registration Code Needed</title>
+    <para>
+    <command>virt-create-rootfs</command> needs a registration
+    code to set up a &productname; root file system.
+    </para>
+   </important>
+   <step>
+    <para>
+     Run the <command>virt-create-rootfs</command> command:
+    </para>
+<screen>virt-create-rootfs --root <replaceable>/PATH/TO/ROOTFS</replaceable> --distro SLES-12.0 -c <replaceable>REGISTRATION_CODE</replaceable></screen>
+   </step>
+   <step>
+    <para>
+     Change the root path to the root file system with the
+     <command>chroot</command> command:
+    </para>
+<screen>chroot <replaceable>/path/to/rootfs</replaceable></screen>
+   </step>
+   <step>
+    <para>
+     Change the password for user &rootuser; with
+     <command>passwd</command>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Create an <systemitem class="username">operator</systemitem> user
+     without &rootuser; privileges:
+    </para>
+<screen>useradd -m operator</screen>
+   </step>
+   <step>
+    <para>
+     Change the operator's password:
+    </para>
+<screen>passwd operator</screen>
+   </step>
+   <step>
+    <para>
+     Leave the chroot environment with <command>exit</command>.
+    </para>
+   </step>
+  </procedure>
+
+  <procedure xml:id="pro.lxc.create-rootfs" os="osuse">
+   <title>Creating a Root File System</title>
+   <para>
+    There is currently no GUI to create a root file system. You will thus need
+    to open a terminal and use <command>zypper</command> as user &rootuser; to
+    populate the new root file system. In the following steps, the new
+    &rootuser; file system will be created in
+    <replaceable>/PATH/TO/ROOTFS</replaceable>.
+   </para>
+   <step>
+    <para>
+     Add the &productname; repository and the corresponding update repository
+     to the new &rootuser; file system:
+    </para>
+    <screen>&prompt.root;zypper --root <replaceable>/PATH/TO/ROOTFS</replaceable> ar http://download.opensuse.org/distribution/leap/42.3/repo/oss/ OSS
+&prompt.root;zypper --root <replaceable>/PATH/TO/ROOTFS</replaceable> ar http://download.opensuse.org/update/leap/42.3/oss/ Update-OSS</screen>
+   </step>
+   <step>
+    <para>
+     Refresh the repositories:
+    </para>
+    <screen>&prompt.root;zypper --root <replaceable>/PATH/TO/ROOTFS</replaceable> ref</screen>
+   </step>
+   <step>
+    <para>
+     Install a minimal system:
+    </para>
+    <screen>&prompt.root;zypper --root <replaceable>/PATH/TO/ROOTFS</replaceable> in -t pattern minimal_base</screen>
+   </step>
+   <step>
+    <para>
+     Set the &rootuser; password:
+    </para>
+    <screen>&prompt.root;echo "ttyS0" >>/PATH/TO/ROOTFS/etc/securetty
+&prompt.root;echo "root:YOURPASSWD" | chpasswd -R /PATH/TO/ROOTFS</screen>
```

Comment:
&prompt.root;echo "root:YOURPASSWD" | chpasswd -R \<replaceable>/PATH/TO/ROOTFS\</replaceable>\</screen>
<replaceable> tag is missing

Reply:
Fixed with 4fef835